mewarpx/mewarpx/poisson_pseudo_1d.py

```diff
@@ -133,11 +132,9 @@ def _run_solve(self):
             return
 
         # get rho from WarpX
-        self.rho_data = mwxrun.get_gathered_rho_grid()
-        # run superLU solver only on the root processor
-        if mwxrun.me == 0:
-            self.rho_data = self.rho_data[0][:,:,0]
-            self.solve()
+        self.rho_data = mwxrun.get_gathered_rho_grid()[:,:,0]
+        # run superLU solver to get phi
+        self.solve()
```

Comment:
There's probably some performance tradeoff here of replicated vs centralized solve calls. The direct solver is such a tiny fraction of execution in the runs we've seen that it probably will never matter, though.